app/controllers/course/lesson_plan_items_controller.rb

```diff
@@ -1,11 +1,16 @@
 class Course::LessonPlanItemsController < Course::ComponentController
   before_action :load_lesson_plan_items, only: [:index]
+  # This can only be done with Bullet once Rails supports polymorphic +inverse_of+.
+  around_action :without_bullet, only: [:index]
```

Comment:
I've modified the without_bullet method to work even when Bullet is not installed.

I wonder if we should be running Bullet in production though...